astrocut/tests/test_cube_cut.py

```diff
@@ -243,6 +243,42 @@ def test_header_keywords_quality(cube_file, ffi_type, tmp_path):
         assert ext1_header['BJDREFI'] == ext2_header['BJDREFI'] == primary_header['BJDREFI']
 
 
+@pytest.mark.parametrize("ffi_type", ["SPOC", "TICA"])
+def test_header_keywords_diffs(cube_file, ffi_type, tmp_path):
+    """Test known header keywords differences"""
+
+    tmpdir = str(tmp_path)
+
+    cutout_maker = CutoutFactory()
+    coord = "256.88 6.38"
+    cutout_size = [5, 3]
+    out_file = cutout_maker.cube_cut(
+        cube_file, coord, cutout_size, ffi_type, output_path=path.join(tmpdir, "out_dir"), verbose=False
+    )
+
+    with fits.open(out_file) as hdulist:
+
+        # Get units from BinTableHDU
+        cols = hdulist[1].columns.info('name, unit', output=False)
+        cols_dict = dict(zip(*cols.values()))
+
+        ncols = 12 if ffi_type == 'SPOC' else 11
+        assert len(cols_dict) == ncols
+
+        # Checking for known keyword differences
+        if ffi_type == 'SPOC':
+            assert hdulist[0].header['TIMEREF'] == 'SOLARSYSTEM', 'TIMEREF keyword does not match expected'
+            assert hdulist[0].header['TASSIGN'] == 'SPACECRAFT', 'TASSIGN keyword does not match expected'
+            assert cols_dict['FLUX'] == 'e-/s', f'Expected `FLUX` units of "e-/s", got units of "{cols_dict["FLUX"]}"'
+            assert np.mean(hdulist[1].data.field('CADENCENO')) == 0.0
+
+        if ffi_type == 'TICA':
+            assert hdulist[0].header['TIMEREF'] is None, 'TIMEREF keyword does not match expected'
+            assert hdulist[0].header['TASSIGN'] is None, 'TASSIGN keyword does not match expected'
+            assert cols_dict['FLUX'] == 'e-', f'Expected `FLUX` units of "e-", got units of "{cols_dict["FLUX"]}"'
+            assert np.mean(hdulist[1].data.field('CADENCENO')) != 0.0
```

Comment:
Are we certain that this `CADENCEN0` will always be a non-zero value for TICA?

Reply:
Yes, the `CADENCE` keyword in the TICA FFI headers (which matches the `FFIINDEX` in the SPOC headers) is the number of FFIs since the start of data collection. This number increments by 1 with each FFI observed, so it will also be positive and non-zero.

I'm going to make a small adjustment to make it clear that `CADENCENO` for TICA should always be non-zero with
`assert hdulist[1].data.field('CADENCENO').all() != 0.0`

Reply:
Thank you!